src/test/java/com/ahaidychuk/dependencyversionupdater/service/RepoServiceTest.java

```diff
@@ -0,0 +1,59 @@
+package com.ahaidychuk.dependencyversionupdater.service;
+
+import com.ahaidychuk.dependencyversionupdater.model.WebHookPayload;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.io.BufferedInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest()
```

Comment:
do you really want to make it SpringBootTest that will start the whole context? Do you really need the whole context here?

Reply:
Yes, you're right, at this stage of development for this class is redundant. Updated